### ISSUE TITLE
Source view is not working

### ISSUE THREAD

**Issue by filips123** [phpDocumentor/phpDocumentor2#1998]
Source view is not working in phpDocumentor3, but it works in phpDocumentor2.

## Expected Behavior

Source view should show source code of the file.

## Actual Behavior

Source view shows blank two lines. 

## Steps to Reproduce the Problem

  1. Create few files with docBlocks.
  2. Create documentation with phpDoc 2.
  3. Create documentation with phpDoc 3.
  4. Open source view of file.

## Your environment

* Version used: v3.0.0-alpha.2
* Install method: PHAR
* PHP version: 7.2.6
* Operating system and version: Ubuntu 16.04 on [Codenvy](https://codenvy.io/) PHP stack

## Screenshots
**phpDocumentor2:**
![phpdoc2](https://user-images.githubusercontent.com/16626308/41913590-00275e66-7951-11e8-8c2d-8212877a4a86.png)

**phpDocumentor3:**
![phpdoc3](https://user-images.githubusercontent.com/16626308/41913616-10bedc86-7951-11e8-8f73-490331815724.png)

**Comment by jaapio** [comment]
It should work when you provide the `--source` flag. This was a bug in phpdoc v2  that always added the source to the docs. 

Please let me know if this works for you.

**Comment by filips123** [comment]
I currently can't test this. I will test this on next Thursday.

**Comment by jaapio** [comment]
I just checked it myself, the `--sourcecode` flag is not working correctly right now. To fix an other issue I enabled it by default like it was done in phpdoc 2. I will leave this issue open to be sure that we fix this issue in the future the right way. 

The latest nightly build will contain the fix and help you forward.

**Comment by mussaweralyasah** [comment]
@jaapio how did you enabled it by default. i am new to this i want the source code to be in the documentation.
right now clicking on the button of source code shows 
✖ Error: File does not exist or is empty
i want it to show the source code in documentation

**Comment by PauchardThomas** [comment]
Same problem.

> ✖ Error: File does not exist or is empty

* Env : Windows
* Version : v3.0.0-alpha.3
* PHP : 7.2.6
* Command : `phpDocumentor run -d src/ -t doc/  --template="clean" --sourcecode`

**Comment by mvriel** [comment]
This issue is resolved on develop. You can now use the `--sourcecode` flag or the `include-source` configuration option to enable this feature. It is disabled by default.

I have also adjusted the default template (called 'clean') to not show the button to open up the source view. Previously it was always there and showed an empty result when you tried to use it.

Please also keep in mind that opening the source view from `file://` is not possible because it is loaded asynchronously and browsers block asynchronous requests from `file://`

**Comment by mvriel** [comment]
I will close this issue for now; if the issue persists for you, please add an update to the issue and we will reopen it

**Comment by oliveratgithub** [comment]
> or the `include-source` configuration option to enable this feature.

@mvriel can you give an example how exactly I'd have to add this to a custom XML-configuration file for phpDocumentor? Thanks

**Comment by MarabyAidara** [comment]
tried `php tools/phpDocumentor --sourcecode -d <path-to-my-symfony-project>  -t <path-to-folder-output>`
.
The doc is generated with the classes and methods but still I can't view the source code  .
I am using phpDocumentor `v3.0.0-beta` with  php 7.3.
Any help is welcome !

**Comment by oliveratgithub** [comment]
@MarabyAidara I got it working using

* v3.0.0-rc (not beta)
* using the „clean“-template (not default)

**Comment by WebDevLan4You** [comment]
Hi @ all
Source code won't work for me with v3.0.0-rc phpDocumentor.phar file and the "clean" template

Some ideas?

![error-source-code-clean template](https://user-images.githubusercontent.com/22098517/87022423-c2961800-c1d6-11ea-9fd8-e82a1e84d41c.png)
1 = files folder for sourcecode view
2 = strange folder generation

The problem exist in 2017 with phpDocumentor 2.0. I fixed the problem, but today the source changed and i am not sure where to go to fix it. the old file was "phpdocumentor/plugin/Twig/Writer/Twig.php". But what I changed, i forgot it.

Help needed. thanks in advance
